doc/concepts/decor.rst

```diff
@@ -166,7 +167,24 @@ Take for example the built-in mechanism for passive leaky dynamics:
 
 .. _cablecell-ions:
 
-4. Ion species
+.. _cablecell-scaled-mechs:
+
+4. Scaled mechanisms
+~~~~~~~~~~~~~~~~~~~~~
+:ref:`Inhomogenous expressions  <labels-iexpr>` can be applied to mechanisms, to scale parameters on a cable cell.
```

Comment:
Hey, a sentence or two on why and how this is desirable would go a long way. Something like:

> It is useful to have a quantity smoothly varying along the dendrite, for example with the distance from the soma. 

Then use this idea in your examples.

Reply:
I've added your suggested sentence and changed to example to a distance expression.